File: lang/php/src/vac.fun/util/SequenceBuilder.php
<?php
    namespace vac\fun\util\SequenceBuilder;

    require_once "vac.fun/util/RandomFactory.php";
    require_once "vac.fun/util/Sequences.php";

    use vac\fun\util\RandomFactory;
    use vac\fun\util\Sequences;

    function packIncreasing(array &$arr): void {
        RandomFactory\seed();
        $arr[0] = RandomFactory\integerN(1, 3);
        for ($i = 1, $length = count($arr); $i < $length; $i++) {
            $arr[$i] = $arr[$i - 1] + RandomFactory\integerN(1, 3);
        }
    }

    function packRandom(array &$arr): void {
        RandomFactory\seed();
        foreach ($arr as &$v) {
            $v = RandomFactory\generateInteger();
        }
    }

    function packDecreasing(array &$arr): void {
        packIncreasing($arr);
        Sequences.reverse($arr);
    }
?>

File: go/muse/util/SequenceBuilder/SequenceBuilder.go
package SequenceBuilder

import (
    "muse/util/RandomFactory"
    "muse/util/Sequences"
)

func PackIncreasing(arr []int) {
    arr[0] = RandomFactory.GenIntN(1, 3)
    for i, length := 1, len(arr); i < length; i++ {
        arr[i] = arr[i - 1] + RandomFactory.GenIntN(1, 3)
    }
}

func PackRandom(arr []int) {
    for i := range arr {
        arr[i] = RandomFactory.GenInt()
    }
}

func PackDecreasing(arr []int) {
    PackIncreasing(arr)
    Sequences.Reverse(arr)
}
